Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_preferences.py

```diff
@@ -282,4 +318,4 @@ def unregister():
     is_meshsync_in_manifest: bpy.props.BoolProperty(name = "Is Meshsync added in the package manifest", default = False)
     is_meshsync_in_manifest_lock: bpy.props.BoolProperty(name= "Is Meshsync resolved by the unity editor", default = False, update = on_in_package_lock_updated)
```

Comment:
Assuming this is user facing we probably want to capitalize Unity Editor

Reply:
Good call, done!